VK/VK/AnimatedViews/Loader.swift

```diff
@@ -0,0 +1,106 @@
+// Loader.swift
+// Copyright © RoadMap. All rights reserved.
+
+// CommonGroupTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+import UIKit
+
+/// asdasd
```

Comment:
```suggestion
/// asdasd
```
Это что за описание? xD